data/i18n/bn/bn.toml

```diff
@@ -1,26 +1,50 @@
 # i18n strings for the Bengali (main) site.
-# NOTE: Please keep the entries in alphabetical order when editing
+# NOTE: Please keep the entries in alphabetical order (english alphabetical order)  when editing
+
+# For "and", see [conjunction_1]
+
+[api_reference_title]
+other = "API রেফারেন্স"
 
 [auto_generated_edit_notice]
 other = "(auto-generated page)"
 
 [auto_generated_pageinfo]
-other = """<p>এই পৃষ্ঠাটি স্বয়ংক্রিয়ভাবে তৈরি হয়।</p><p>আপনি যদি এই পৃষ্ঠার সাথে একটি সমস্যা রিপোর্ট করার পরিকল্পনা করেন, তাহলে উল্লেখ করুন যে পৃষ্ঠাটি আপনার সমস্যার বিবরণে স্বয়ংক্রিয়ভাবে তৈরি হয়েছে। Kubernetes প্রজেক্টের অন্য কোথাও ঠিক করার প্রয়োজন হতে পারে।</p>"""
+other = """<p>এই পৃষ্ঠাটি স্বয়ংক্রিয়ভাবে তৈরি হয়।</p><p>আপনি যদি এই পৃষ্ঠার সাথে একটি সমস্যা রিপোর্ট করার পরিকল্পনা করেন, তাহলে উল্লেখ করুন যে পৃষ্ঠাটি আপনার সমস্যার বিবরণে স্বয়ংক্রিয়ভাবে তৈরি হয়েছে। কুবারনেটিস প্রজেক্টের অন্য কোথাও ঠিক করার প্রয়োজন হতে পারে।</p>"""
+
+[blog_post_show_more]
+other = "আরো পোস্ট দেখান..."
+
+[banner_acknowledgement]
+other = "এই নোটিস লুকান"
+
+[case_study_prefix]
+other = "কেস স্টাডিজ:"
 
 [caution]
 other = "সতর্কতা:"
 
 [cleanup_heading]
 other = "ক্লিনিং"
 
+# Localization note (no need to translate this note)
+# This is always followed by Chinese text, as required by the Kubernetes
+# project's ICP license, issued by the government of the People's Republic
+# of China.
```

Comment:
```suggestion
```

Reply:
Can we please keep this? It will help us maintain a certain line break, which is further helpful for updating this file.

Reply:
Sure, it's fine to keep it.